tensorflow/core/transforms/remapper/pass.cc

```diff
@@ -119,29 +333,38 @@ class Remapper : public RemapperBase<Remapper> {
       populateRemapperPDLLPatterns(patterns);
       return;
     }
-    if (enable_mkl_patterns_) {
+    if (enable_onednn_patterns_) {
       patterns.insert<MatchMulSigmoid>(context);
       // TODO(chiahungduan): Currently, the only pattern implemented in PDLL is
       // the same one as `MatchMulSigmoid`. Remove the one of them when there's
       // a decision that which one is preferred.
       populateRemapperPDLLPatterns(patterns);
     }
+    patterns.insert<ContractionBiasAddRewriter>(*helper_);
+    // Insert multiple pattern rewriters from template instantiations by
+    // activation ops.
+    InsertPatterns<ContractionBiasAddActivationRewriter, OpKind::Relu,
+                   OpKind::Relu6, OpKind::Elu, OpKind::LeakyRelu, OpKind::Tanh,
+                   OpKind::Sigmoid>(patterns, *helper_);
   }
 
   void populateRemapperPDLLPatterns(RewritePatternSet &patterns) {
     mkl::populateGeneratedPDLLPatterns(patterns);
   }
 
   FrozenRewritePatternSet final_patterns_;
+  std::shared_ptr<OpPropertyHelper> helper_;
```

Comment:
Can we make OpPropertyHelper a value based type and copy it here and everywhere (instead of keeping a reference)? It shouldn't be more expensive than a shared_ptr really?

Reply:
Done.